detectable/src/main/java/com/synopsys/integration/detectable/detectables/projectinspector/ProjectInspectorExtractor.java

```diff
@@ -31,6 +28,12 @@ public Extraction extract(
         ExecutableTarget inspector
     ) throws ExecutableFailedException {
         File outputFile = new File(outputDirectory, "inspection.json");
+        boolean includeShadedDependencies = false;
+
+        if(extra.contains("include_shaded_dependencies")) {
```

Comment:
This section is essentially to know if this method was invoked for just including the shaded dependencies.